efi/meson.build

```diff
@@ -222,7 +224,11 @@ app = custom_target(efi_name,
                       '--os', host_machine.system(),
                       '--objcopy', objcopy,
                       '--genpeimg', genpeimg.found() ? genpeimg : ''
-                    ],
+                    ]
+if host_cpu == 'arm' or (objcopy_version.version_compare ('< 2.38') and host_cpu == 'aarch64')
```

Comment:
I think i'd rather see this block combined with the other one in line 148-149.  You can declare an empty array for the arguments for this command outside of it.  something like this:
```
sbat_extra = []
if host_cpu == ....
    sbat_extra = ['--objcopy-manual-symbols']
.
.
.
command : [
.
.
]+ sbat_extra
```